pkgs/development/python-modules/prov/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, lxml
+, networkx
+, dateutil
+, six
+, pydotplus
+, rdflib
+}:
+
+buildPythonPackage rec {
+  name = "${pname}-${version}";
+  pname = "prov";
+  version = "1.5.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1vkpayns0mf7i2357vxyvi2n5h5xsyg56ik2yqgrc91k3gx4x9wn";
+  };
+
+  propagatedBuildInputs = [
+    lxml
+    networkx
+    dateutil
+    six
+    pydotplus
+    rdflib
+  ];
+  doCheck = false; # takes forever!
```

Comment:
What is "forever"? Test suites of most scientific python packages take some time.

Reply:
Looking into this now (I wrote some of these expressions a number of months ago).

Reply:
```
Ran 956 tests in 3594.012s

FAILED (failures=22, expected failures=17)
Test failed: <unittest.runner.TextTestResult run=956 errors=0 failures=22>
```

So about an hour (total build time was about an hour and 10).

I am investigating whether the errors are present in 1.5.1 before reporting upstream. (NB: Nipype currently requires exactly 1.5.0, this is being dealt with)